apps/web/components/Organization/Wrapper/Themes/FiskistofaTheme/FiskistofaFooter.tsx

```diff
@@ -17,6 +17,7 @@ import { webRichText } from '@island.is/web/utils/richText'
 import { useNamespace } from '@island.is/web/hooks'
 
 import * as styles from './FiskistofaFooter.css'
+import { ReactNode } from 'react'
```

Comment:
It'd be nice if the styles import would stay at the bottom, so maybe adding this to the top of the file would look better

Reply:
Sure :+1: